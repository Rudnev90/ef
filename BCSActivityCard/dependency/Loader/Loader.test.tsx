
import * as React from 'react';

import renderer from 'react-test-renderer';

import { ActivityTabLoader } from '.';

describe('ActivityTabLoader', () => {
  it.concurrent('ActivityTabLoader render ok', async () => {
    const output = renderer.create(<ActivityTabLoader classes={{}} theme={null} />).toJSON();
    expect(output).toMatchSnapshot();
  });
});
