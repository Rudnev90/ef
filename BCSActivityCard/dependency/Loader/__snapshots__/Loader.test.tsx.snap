// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="MuiPaper-root-1 MuiPaper-elevation2-5 MuiPaper-rounded-2"
>
  <section>
    <div>
      <div
        className="undefined animated"
      />
    </div>
    <div>
      <div>
        <div
          className="undefined animated"
        />
      </div>
      <div>
        <div>
          <div
            className="undefined animated"
          />
        </div>
      </div>
    </div>
    <div>
      <div
        className="undefined animated"
      />
    </div>
  </section>
</div>
`;
